lib/ui/views/farm_list/layouts/components/farm_refresh_icon.dart

```diff
@@ -38,9 +48,11 @@ class _FarmRefreshIconState extends ConsumerState<FarmRefreshIcon> {
             isRefreshSuccess = true;
           },
         );
-        final farmListItemState =
-            context.findAncestorStateOfType<FarmListItemState>();
-        await farmListItemState?.reload();
+
+        // ignore: unused_result
+        await ref.refresh(farmProvider.future);
```

Comment:
need explanation. difference with invalidate

Reply:
`refresh` is equivalent to `validate` then `read` : it will then re-evaluate the provider now and return its new value.

I used `refresh` here to keep behavior unchanged.